services/token_create.proto

```diff
@@ -191,4 +191,9 @@ message TokenCreateTransactionBody {
      * The custom fees to be assessed during a CryptoTransfer that transfers units of this token
      */
     repeated CustomFee custom_fees = 21;
+
+    /**
+     * The Key which can pause and unpause the Token. If Empty the token is unpaused by default.
+     */
```

Comment:
1. What is the behavior if the key is set?  I assume `token.paused` will default to `Unpaused`.  Should we allow a token to be created where it defaults to `Paused`, similar to how `freezeDefault` works?  Either way, it would be good to mention the default here.
2. Update comment as Empty now defaults to `PauseNotApplicable`

Reply:
Token when created will be `Unpaused` if pauseKey is set and `PauseNotApplicable` if it is not. 
Mentioned the defaults.

Reply:
We currently dont want to allow a token to be paused by default.

Reply:
Looks good, and makes sense.